libkbfs/config_local.go

```diff
@@ -208,6 +209,15 @@ func MakeLocalUsers(users []libkb.NormalizedUsername) []LocalUser {
 	return localUsers
 }
 
+func getDefaultCleanBlockCacheCapacity() uint64 {
+	vmstat, err := mem.VirtualMemory()
+	if err == nil {
+		// By default,  we try to cap at 1/8 of available memory
+		return vmstat.Total / 8
```

Comment:
Got you; I think it's reasonable. If use wants to increase to cap, she can always set it from commandline. Since we are capping with `MaxBlockSizeBytesDefault * 1024` too, I'm changing `/8` to `/4` to loosen up the lower-end. Let me know if you think it's too big!